extensions/amp-story/0.1/system-layer.js

```diff
@@ -0,0 +1,258 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {EventType, dispatch} from './events';
+import {dev} from '../../../src/log';
+import {Services} from '../../../src/services';
+import {ProgressBar} from './progress-bar';
+
+
+// TODO(alanorozco): Use a precompiled template for performance
+/*eslint-disable max-len */
+const TEMPLATE =
+    `<div class="i-amphtml-story-ui-right">
```

Comment:
Let's use single quote here to make sure it's static. Also type with `/** @private @const {string} */`.

Reply:
Done